app/main/db/queries.py

```diff
@@ -560,3 +560,46 @@ def get_all_transferring_bodies():
     for body in Body.query.all():
         bodies.append(body.Name)
     return bodies
+
+
+def get_breadcrumb_values(
+    transferring_body_id=None, series_id=None, consignment_id=None
+):
+    dict_values = {}
+    if transferring_body_id:
+        transferring_body = (
+            Body.query.filter(Body.BodyId == transferring_body_id).first()
```

Comment:
Since BodyId is the primary key for the Body Table, it is unique and so I think we can just do

```suggestion
            Body.query.get(transferring_body_id)
```

Reply:
updated logic to follow above